From 7b03dcee291e134ac4a1c4fc7d122af3f0d7739e Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Wed, 1 Jun 2022 16:14:14 +0800
Subject: [PATCH 14/31] Revert "drm/msm/dsi: Use connector directly in
 msm_dsi_manager_connector_init()"

This reverts commit 1ac7c6d75ede1dc4eddbe4da9de9810506ed8af1.
---
 drivers/gpu/drm/msm/dsi/dsi_manager.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/msm/dsi/dsi_manager.c b/drivers/gpu/drm/msm/dsi/dsi_manager.c
index f3ff2cdc288b..73127948f54d 100644
--- a/drivers/gpu/drm/msm/dsi/dsi_manager.c
+++ b/drivers/gpu/drm/msm/dsi/dsi_manager.c
@@ -625,7 +625,7 @@ struct drm_connector *msm_dsi_manager_connector_init(u8 id)
 	return connector;
 
 fail:
-	connector->funcs->destroy(connector);
+	connector->funcs->destroy(msm_dsi->connector);
 	return ERR_PTR(ret);
 }
 
-- 
2.27.0

